site/content/entry/yuzu-progress-report-jun-2023/index.md

```diff
@@ -0,0 +1,735 @@
+Something to keep in mind if you’re exploring Aionios with an NVIDIA GPU.
+
+### AMD
+
+Since driver release 23.7.1, a mysterious older “23.10.01.41 for Additional Vulkan Extensions” version, and the beta driver only available to Windows Insider testers, AMD official Vulkan drivers introduced a regression in how the VK_EXT_extended_dynamic_state3 extension handles colour blending.
+[Your writer](https://github.com/goldenx86) {{< gh-hovercard "10946" "blocked the affected bit" >}} while we monitor if future AMD driver releases manage to solve this problem.
+
+{{< single-title-imgs-compare
+	"Who gave Link the Batman’s Detective Mode? (The Legend of Zelda: Tears of the Kingdom)"
+	"./amdbug.png"
+	"./amdfix.png"
+>}}
+
+The performance loss from this change is very small, so feel free to update to the latest drivers.
+
+[GPUCode](https://github.com/GPUCode) has found a way to emulate D24 support on AMD GPUs by using the `VK_EXT_depth_bias_control` extension, which is currently only available on the RADV Mesa driver on Linux.
+In the near future, we’ll check if this method solves the remaining graphical issues specific to AMD hardware. 
+If it works, it would also solve the issue on the AMD official drivers, once/if the extension is supported there too.
+
+We know it’s not fun having to switch to OpenGL to be able to play some games, we’re ready to combat even hardware limitations to defeat this foe.
+
+### Intel
+
+Intel got its fair share of fixes this month too, first of all, as promised last month, byte[] added a translation for FP64 shaders, to turn them into FP32 ones.
+This was required because Generation 12 Intel graphics (UHD 700/Iris Xe/Arc A series) removed support on a hardware level for double precision operations.
+With this {{< gh-hovercard "10693" "shader translation" >}} in place, games will no longer suddenly crash when trying to run very high precision shaders, for example during the opening cutscene of `Tears of the Kingdom` when starting a new game.
+
+The next issue is an interesting mix of unique behaviour by the Nintendo Switch, yuzu’s code at fault, and another case of lack of hardware support. 
+Intel defined the use of barriers as specified by SPIR-V, but the Switch’s Tegra X1 (and interestingly, any other GPU vendor besides Intel, including Android ones) prefers to follow the PTX ISA documentation for barriers, which defines what to do if no checks for existing threads are done.
+
+By {{< gh-hovercard "10699" "removing barriers" >}} in the conditional control flow, Intel GPUs are free from their device loss crashes.
+
+And lastly, one loose end remained, our toggle to disable the compute pipeline on Intel GPUs.
+Since this option was a stopgap solution until a driver update was released, it has become irrelevant now that Intel took their long time to release a proper fix, so toastUnlimited made the option only show up in yuzu’s settings if an old driver known to be affected is still being run by the user.
```

Comment:
```suggestion
This option was a stopgap solution until a driver update was released. But since Intel has finally published a driver update fixing the issues, it has now become irrelevant. toastUnlimited therefore modified the option to only show up in yuzu’s settings if an old driver known to be affected is still in use.
```